MANIFESTO.md

```diff
@@ -3,37 +3,37 @@ layout: page
 permalink: /
 image: assets/openops.png
 ---
-We – [the founding and supporting organizations](#supporting-organizations) – proclaim transparency, sharing knowledge and are building a community of practice - open operations.
+We – [the founding and supporting organizations](#supporting-organizations) – proclaim our primary objectives to be transparency and the sharing of knowledge, and are in the process of building a community of practice – Open Operations.
```

Comment:
As german native speaker, two "ands" within an enumeration of three sounds strange but if it's proper english okay. 🤔

Reply:
We could replace it with **to be transparency as well as the sharing of knowledge**